README.md

```diff
@@ -27,3 +27,13 @@ components, see the [Metal³ docs](https://github.com/metal3-io/metal3-docs).
 * [Configuration](docs/configuration.md)
 * [Testing](docs/testing.md)
 * [Publishing Images](docs/publishing-images.md)
+
+### Important Notes
+
+Whenever there is a change in `config/` directory, please remember to run the
+following command:
+
+`make manifests`
+
+This will render the `config/render/capm3.yaml`. Please do not change the
+content of `config/render/capm3.yaml` manually.
```

Comment:
Does the `generate` CI job accurately test to ensure this is done? It looks like it checksums the files under `config/` then runs `make generate manifests` and then compares the new checksums with the old. Is it capturing all of the right files?

Reply:
https://prow.apps.ci.metal3.io/view/gcs/metal3-prow-artifacts/pr-logs/pull/metal3-io_baremetal-operator/702/generate/1321414863867613184 shows the job output for this PR

Reply:
Yes does. Its part from the `manifests` make target and as you pointed out it does the `make generate manifests` . One of the PRs tried to modify the `config/render/capm3.yaml` unknowingly. It was caught by generate, [here](https://storage.googleapis.com/metal3-prow-artifacts/pr-logs/pull/metal3-io_baremetal-operator/700/generate/1321372458866970627/build-log.txt) is the output of the failed run

Reply:
OK, great. I wanted to make sure we weren't just relying on the docs to catch this sort of mistake.